### PR TITLE
add appswitcher-server to stack, configure appswitcher frontend component via .env

### PR DESCRIPTION
**Pull request by eidottermihi**
**Description**

- adds [appswitcher-server](https://github.com/it-at-m/appswitcher-server) to local stack, containing "apps" of local stack (pgAdmin, Keycloak)
- configuration of appswitcher-server url is done via `.env` files during frontend buildtime
  - dev build: use local instance from stack
  - production build: use LHM-internal instance

grafik

**Reference**

Issues #100

### CODE REVIEW

**Review by devtobi — CHANGES_REQUESTED** [review]

File: stack/appswitcher-server/application.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+appswitcher.apps:
+  pgadmin:
+    display-name: pgAdmin
+    url: http://localhost:5050
+    image-url: https://upload.wikimedia.org/wikipedia/commons/a/ad/Logo_PostgreSQL.png
```
[the remark this answers is not in the document]
Comment (eidottermihi) [reply stack/appswitcher-server/application.yml:5]: Right now thats not possible with `appswitcher-server`. But as we use those "apps" only for "development" purposes with the local stack I think that's ok.

[the remark this answers is not in the document]
Comment (devtobi) [reply stack/appswitcher-server/application.yml:5]: Ok that's fine for now. Still would recommend creating a follow up issue for the repo of `app switcher-server`.

**Review by devtobi — APPROVED** [review]
LGTM

**Review by devtobi — APPROVED** [review]
LGTM, great that static files are working. :)

**Comment by devtobi** [comment]
Reminder: This PR is also relevant for github.com/it-at-m/refarch

**Pull request merged**